components/sysutils/virtualbox/virtualbox.p5m

```diff
@@ -32,3 +137,108 @@ file path=platform/i86pc/kernel/drv/vboxflt.conf
+file path=usr/share/icons/hicolor/32x32/mimetypes/virtualbox-vhd.png
+file path=usr/share/icons/hicolor/32x32/mimetypes/virtualbox-vmdk.png
+file path=usr/share/icons/hicolor/48x48/mimetypes/virtualbox-hdd.png
+file path=usr/share/icons/hicolor/48x48/mimetypes/virtualbox-ova.png
+file path=usr/share/icons/hicolor/48x48/mimetypes/virtualbox-ovf.png
+file path=usr/share/icons/hicolor/48x48/mimetypes/virtualbox-vbox-extpack.png
+file path=usr/share/icons/hicolor/48x48/mimetypes/virtualbox-vbox.png
+file path=usr/share/icons/hicolor/48x48/mimetypes/virtualbox-vdi.png
+file path=usr/share/icons/hicolor/48x48/mimetypes/virtualbox-vhd.png
+file path=usr/share/icons/hicolor/48x48/mimetypes/virtualbox-vmdk.png
+file path=usr/share/icons/hicolor/64x64/mimetypes/virtualbox-hdd.png
+file path=usr/share/icons/hicolor/64x64/mimetypes/virtualbox-ova.png
+file path=usr/share/icons/hicolor/64x64/mimetypes/virtualbox-ovf.png
+file path=usr/share/icons/hicolor/64x64/mimetypes/virtualbox-vbox-extpack.png
+file path=usr/share/icons/hicolor/64x64/mimetypes/virtualbox-vbox.png
+file path=usr/share/icons/hicolor/64x64/mimetypes/virtualbox-vdi.png
+file path=usr/share/icons/hicolor/64x64/mimetypes/virtualbox-vhd.png
+file path=usr/share/icons/hicolor/64x64/mimetypes/virtualbox-vmdk.png
+file path=usr/share/icons/hicolor/72x72/mimetypes/virtualbox-hdd.png
+file path=usr/share/icons/hicolor/72x72/mimetypes/virtualbox-ova.png
+file path=usr/share/icons/hicolor/72x72/mimetypes/virtualbox-ovf.png
+file path=usr/share/icons/hicolor/72x72/mimetypes/virtualbox-vbox-extpack.png
+file path=usr/share/icons/hicolor/72x72/mimetypes/virtualbox-vbox.png
+file path=usr/share/icons/hicolor/72x72/mimetypes/virtualbox-vdi.png
+file path=usr/share/icons/hicolor/72x72/mimetypes/virtualbox-vhd.png
+file path=usr/share/icons/hicolor/72x72/mimetypes/virtualbox-vmdk.png
+file path=usr/share/icons/hicolor/96x96/mimetypes/virtualbox-hdd.png
+file path=usr/share/icons/hicolor/96x96/mimetypes/virtualbox-ova.png
+file path=usr/share/icons/hicolor/96x96/mimetypes/virtualbox-ovf.png
+file path=usr/share/icons/hicolor/96x96/mimetypes/virtualbox-vbox-extpack.png
+file path=usr/share/icons/hicolor/96x96/mimetypes/virtualbox-vbox.png
+file path=usr/share/icons/hicolor/96x96/mimetypes/virtualbox-vdi.png
+file path=usr/share/icons/hicolor/96x96/mimetypes/virtualbox-vhd.png
+file path=usr/share/icons/hicolor/96x96/mimetypes/virtualbox-vmdk.png
+file path=usr/share/mime-info/virtualbox.keys
+file path=usr/share/mime-info/virtualbox.mime
+file path=usr/share/mime/packages/virtualbox.xml
+file path=usr/share/pixmaps/VBox.png
+file path=usr/share/pixmaps/virtualbox.png
+file path=var/svc/manifest/application/virtualbox/virtualbox-autostart.xml
+file path=var/svc/manifest/application/virtualbox/virtualbox-balloonctrl.xml
+file path=var/svc/manifest/application/virtualbox/virtualbox-zoneaccess.xml
+#file path=vbox.pkginfo
+#file path=vbox.space
+
+driver name=vboxdrv 
```

Comment:
Do we want to have separate driver package or mark drivers as GZ-only content?

Reply:
Problem is that the script for post/pre-install steps assumes that all files are delivered. Would need some time to look at splitting the logic if a different package are provided for drivers.
Should I use a transform to mark drivers are GZ-only content? I have never done this.

Reply:
It does not work, it chokes on the actual driver files and if the same is added to them pkg returns an internal error.

Reply:
Sorry, looking at illumos-gate build receipts, it seems variant.opensolaris.zone=global should be added only to actual files.